lib/src/document.dart

```diff
@@ -10,12 +10,22 @@ import 'inline_parser.dart';
 /// Maintains the context needed to parse a Markdown document.
 class Document {
   final Map<String, LinkReference> linkReferences = <String, LinkReference>{};
-  final ExtensionSet extensionSet;
   final Resolver? linkResolver;
   final Resolver? imageLinkResolver;
   final bool encodeHtml;
+
+  /// Whether to have default block syntaxes.
+  final bool withDefaultBlockSyntaxes;
+
+  /// Whether to have default inline syntaxes.
```

Comment:
I think "use" would be better than "have" here; "Whether to use ..."

Reply:
Fixed